src/modules/ui-action.js

```diff
@@ -0,0 +1,19 @@
+const addItem = (todos, input, render) => {
+  const newItemText = input.value.trim();
+
+  if (newItemText === '') {
+    return;
+  }
+
+  const newTodo = {
+    description: newItemText,
+    completed: false,
+    index: todos.length,
+  };
+
+  todos.push(newTodo);
+  input.value = '';
+  render();
+};
+
+export default addItem;
```

Comment:
Kindly fix this error on your project as it is currently impossible to add a task in your project😉.

<img width="1390" alt="image" src="https://github.com/Edgahkipkemoi/To-do-list/assets/64643995/cdc78469-d654-4d41-87b8-72752fef80c4">



https://github.com/Edgahkipkemoi/To-do-list/assets/64643995/168ba653-f2eb-44c6-a268-41b5ee35daf6